.circleci/config.yml

```diff
@@ -220,6 +220,10 @@ jobs:
       - create-creme-project
       - setup-creme-unit-tests:
           local_settings: 'pgsql_settings'
+      - run:
+          name: "PostgreSQL extensions"
+          command: |
+            echo "SELECT * FROM pg_available_extensions"|creme dbshell --settings=<< pipeline.parameters.instance_directory >>.settings
```

Comment:
Would deserve its own commit I presume.

Reply:
I moved it to [draft/pg_enumerable_unaccent](https://github.com/HybirdCorp/creme_crm/tree/draft/pg_enumerable_unaccent)